src/components/FormLogin/index.tsx

```diff
@@ -90,6 +99,8 @@ const FormLogin = () => {
             <Button
               type="submit"
               width="full"
+              isLoading={formLoginStatus === 'submitting'}
```

Comment:
Should we create a getter `isFormSubmitting` in reducer to get this boolean value?

Reply:
Add `isFormSubmitting` https://github.com/Lahphim/react-surveyor-challenge/pull/65/commits/1cd40768b4933c2d0be2b72bdca73a092b4ce643